listenbrainz/webserver/views/stats_api.py

```diff
@@ -96,29 +91,30 @@ def get_artist(user_name):
     if user is None:
         raise APINotFound("Cannot find user: %s" % user_name)
 
-    stats = db_stats.get_user_artists(user['id'])
-    if stats is None:
+    all_stats: UserArtistStatRow = db_stats.get_user_artists(user['id'])
```

Comment:
is this type needed here if we know the return type of the `get_user_artists` method?